### DISCUSSION TITLE
Stash on linux cant find files

### DISCUSSION THREAD

**Discussion by camerons343** [stashapp/stash#1970] [Q&A] [answered]
Ive tried running it in docker setting volumes and running the standalone version by itself. Both instances they have correct pathing to the files, but when logging into the web page there is no scenes available. Im probably doing something wrong but help would be much appreciated.

**Comment by WithoutPants** [comment, marked as answer]
Have you run a Scan? 

The final page of the setup wizard states:

> When you are satisfied with these settings, you can begin scanning your content into Stash by clicking on Tasks, then Scan.

**Reply by WithoutPants** [reply]
Click on the Settings icon - it should navigate to the tasks page by default.
